packages/interactive_media_ads/README.md

```diff
@@ -4,12 +4,274 @@ Flutter plugin for the [Interactive Media Ads SDKs][1].
 
 [![pub package](https://img.shields.io/pub/v/interactive_media_ads.svg)](https://pub.dev/packages/interactive_media_ads)
 
-A Flutter plugin for using the Interactive Media Ads SDKs on Android and iOS.
+IMA SDKs make it easy to integrate multimedia ads into your websites and apps. IMA SDKs can request
+ads from any [VAST-compliant][2] ad server and manage ad playback in your apps. With IMA client-side
+SDKs, you maintain control of content video playback, while the SDK handles ad playback. Ads play in
+a separate video player positioned on top of the app's content video player.
 
 |             | Android | iOS   |
 |-------------|---------|-------|
 | **Support** | SDK 19+ | 12.0+ |
 
 **This package is still in development.**
 
+## IMA client-side overview
+
+Implementing IMA client-side involves five main SDK components, which are demonstrated in this
+guide:
+
+* [AdDisplayContainer][3]: A container object where ads are rendered.
+* [AdsLoader][4]: Requests ads and handles events from ads request responses. You should only
+instantiate one ads loader, which can be reused throughout the life of the application.
+* [AdsRequest][5]: An object that defines an ads request. Ads requests specify the URL for the VAST
+ad tag, as well as additional parameters, such as ad dimensions.
+* [AdsManager][6]: Contains the response to the ads request, controls ad playback,
+and listens for ad events fired by the SDK.
+* [AdsManagerDelegate][8]: Handles ad events and errors that occur during ad or stream
+initialization and playback.
+
+## Usage
+
+This guide demonstrates how to integrate the IMA SDK into a new `Widget` using the [video_player][7]
+plugin to display content.
+
+1. Add Android Required Permissions
+
+If building on Android, add the user permissions required by the IMA SDK for requesting ads in
```

Comment:
If you are going to use a bulleted list for the headings, I would make all of the content part of the list (by indenting *everything* in this entire section by two spaces), so that the content of each section isn't offset further left than its heading.

Alternatively, you could do `### 1. Add Android Required Permissions`, etc.